Chaap/Chaap/Presentation/Chaap/ChaapComposeView.swift

```diff
@@ -232,22 +247,25 @@ struct ChaapComposeView: View {
                     .font(.chBodyRegular)
                     .lineHeight(1.4, fontSize: 18)
                     .foregroundStyle(Color.chLabelWhiteSecondary)
+                    .frame(maxWidth: .infinity, alignment: .center)
             }
             
             TextEditor(text: $chaap.memo)
+                .maxLength(text: $chaap.memo, 70)
                 .font(.chBodyRegular)
                 .lineHeight(1.4, fontSize: 18)
                 .foregroundStyle(Color.chLabelWhitePrimary)
-                .frame(height: 130)
                 .scrollContentBackground(.hidden)
                 .scrollDisabled(true)
                 .background(Color.clear)
                 .autocorrectionDisabled(true)
                 .textInputAutocapitalization(.never)
                 .multilineTextAlignment(.center)
-                .lineLimit(3)
                 .tint(Color.chLabelWhitePrimary)
+                .focused($isFocused)
         }
+        .padding(.horizontal, 15)
+        .padding(.vertical, 1)
```

Comment:
요 부분 확인해보니
```suggestion
        .padding(.horizontal, 20)
        .padding(.vertical, 16)
```
디자인에는 위와 같이 되어 있는데 요것도 실제 화면 보고 맞추신 걸까요?!

Reply:
그거는 피그마 보고 했더니 이상하게 더 크게 들어가서 눈대중으로 비슷하게 맞췄어요